portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -60,11 +60,12 @@
    */
   private static String validateIncomingComment(String commentAuthor, String commentBody) {
     List<String> validationErrors = new ArrayList<String>();
-
-    if (commentAuthor.isBlank()) {
+    
+    // Used in place of string.isBlank(), which is only available in Java 11
+    if (commentAuthor.length() == 0 || commentAuthor.chars().allMatch(Character::isWhitespace)) {
```

Comment:
What about trimming the string first? Like `commentAuthor.trim().length == 0`?

Otherwise considering this logic is used again immediately below it should probably be its own utility function.

Reply:
Good point, and functionally I'd want the inputs to be trimmed before they enter the datastore anyways. Changing now.